### PR TITLE
fix window's parallel issue

### LINKED ISSUE #7: Fix parallel runs on Windows
Currently the `Simulation` object is created from a fully instantiated module, which creates issues when running parallel on windows (Linux allows for pickling of objects, but Windows does not: currently, objects - i.e. simulation modules - are pickled when they are passed to workers in `multiprocessing`). 

I believe a straight-forward solution would be to just grab the name of the module in `Simulation.from_module`, and only store the path of the module. For the actual simulation, the module can be re-loaded in `Simulation.run`.

Supersedes #3

### PR DESCRIPTION
**Pull request by DavidAkinpelu**
Fixed Window's parallel issue.

Fixes #7.

### CODE REVIEW

**Comment by ischoegl** [comment]
Looks good. Let’s do one pull request per issue. ... could you have a quick look at unit tests?

**Comment by DavidAkinpelu** [comment]
Okay

File: ctwrap/simulation.py [hunk 1]
```diff
@@ -21,25 +22,25 @@
...
-
     Required module functions 'run' and 'defaults' are passed through
     """
 
     def __init__(self, module, output=None):
         """Constructor for `Simulation` object.
-
         Arguments:
-           module (module): handle to module running the simulation
+           module (module):  handle name or handle to module running the simulation
```
Comment (ischoegl) [inline ctwrap/simulation.py:33]: call this: "module or name of module running the simulation"
I think the pep 484 convention would be `Union[object, str]`

**Review by ischoegl — CHANGES_REQUESTED** [review]
Minor comment ... looks ok otherwise. Unit tests still fail on my Windows system.

**Pull request merged**